packages/viewer/src/components/viewer-pin-tool/viewer-pin-tool.tsx

```diff
@@ -60,6 +60,18 @@ export class ViewerPinTool {
   @Prop({ mutable: true })
   public mode: ViewerPinToolMode = 'view';
 
+  /**
+   * The primary color for new pins.
+   */
```

Comment:
Thoughts on expanding what a "primary" color for these pins is? I'm not sure I'd understand what these actually change from documentation currently. (or if this should be internal, could we add the `@internal`?)

Reply:
This is not internal, this is currently used for setting the primary/accent color on a per pin basis. I'll expand on this in a comment.